cs/libdpr/src/FASTER.libdpr/dprfinder/resp/RespUtil.cs

```diff
@@ -1,11 +1,22 @@
 using System;
 using System.Collections.Generic;
+using System.Net;
 using System.Linq;
 
 namespace FASTER.libdpr
 {
     public static class RespUtil
     {
+        private static readonly Dictionary<string, int> typeToInt = new Dictionary<string, int>
```

Comment:
Let's just use strings directly for cluster type unless there's a performance reason?

Reply:
Added a TODO for this. I agree that we should do this, and I will do this later, but I don't want to deal with the whole communication setup at the moment. The only reason I chose an int is because it has a constant size and it was more straightforward to send it that way.